src/meta/processors/job/JobExecutor.h

```diff
@@ -22,32 +22,74 @@ class JobExecutor {
   explicit JobExecutor(kvstore::KVStore* kv) : kvstore_(kv) {}
   virtual ~JobExecutor() = default;
 
-  // Check the arguments about the job.
+  /**
+   * @brief Check the arguments about the job.
```

Comment:
If the comment start with capital letter, then all comments should keep consistent? ps. And I think make first letter capital is better.

Reply:
done